docs/vscode-template/tasks.json

```diff
@@ -0,0 +1,41 @@
+{
+  // See https://go.microsoft.com/fwlink/?LinkId=733558
+  // for the documentation about the tasks.json format
+  "version": "2.0.0",
+  "options": {"shell": {"executable": "bash", "args": ["-lic"]}},
+  "tasks": [
+    {
+      "label": "compile (maven)",
+      "type": "shell",
+      "command": [
+        "mvn",
+        "compile",
+        "-Dpmd.skip=true", // without pmd check
+        "-Dcheckstyle.skip=true" // without style check
```

Comment:
I know this will help the dev cycle be faster, but my concern is that it will lead to PRs that fail build due to style checks.  We'll need to communicate process on building with checks locally before pushing.

Reply:
Yes, I agree. For PRs devs should at the very least do `mvn clean package` and `docker build` and `docker run`.

Although these will only affect the build process within VSCode, we can remove them if a development/production consistency is desired for debug as well.

What is your thought? Should we remove the pmd and style skips?

Reply:
Let's leave it this way and try communicating the pre-PR process to everyone.  If we start having linter issues checked in we'll revert.